queries/hjson/highlights.scm

```diff
@@ -0,0 +1,16 @@
+(true) @boolean
+(false) @boolean
+(null) @constant.builtin
+(number) @number
+(pair key: (string) @label)
+(pair value: (string) @string)
+(array (string) @string)
+;  (string_content (escape_sequence) @string.escape)
```

Comment:
are these not used or do you plan to support them?

Reply:
I am still figuring out how to handle quoteless string and the escape sequence, I do plan to support them and as everything other than that is usable now, I have just commented it out there.